src/components/button/button.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+const Button = ({ className, onClick, disabled, children, text }) => {
+
+  return (
+    <button disabled={disabled} className={className} onClick={onClick} >
+      {children}
+      {text}
+    </button>
+  )
+};
```

Comment:
I wanted to make a base component and use it in different places, because I have many buttons with different functions. But, in this PR, I only had one button.

Reply:
I meant, this component is the same as an HTML button, you didn't change anything between your component and the HTML button. Defining a component that the only thing it makes is passing the prop to the HTML button isn't a good way. If you want to define a common component you need to customize something like styles, actions (onClick, onChange,..), labels, etc. But you not change any think is unnecessary define a new component

Reply:
But, that Button at the mobile-menu file, for example, accepts two different icons, I add different styles and a function.